packages/components/src/local/organisms/support-panel/types/props.d.ts

```diff
@@ -1,30 +1,28 @@
 import { TurnFormats } from '@serge/config'
-import { ChannelPlanning, ForceData, MessagePlanning, Role } from '@serge/custom-types'
+import { ChannelPlanning, ForceData, MessagePlanning, PlayerUi, Role } from '@serge/custom-types'
 
 export default interface PropTypes {
   messages: MessagePlanning[]
   turnPresentation?: TurnFormats
-  hideForcesInChannel: boolean
-  selectedForce: ForceData['uniqid']
-  selectedRole: Role['roleId']
   forceIcons: any[]
   forceNames: string[]
-  forces: Force[]
   onRead: (message: MessagePlanning) => void
   onUnread: (message: MessageChannel | ChatMessage) => void
   onReadAll: () => void
   /**
    * definition of this channel
    */
   channel: ChannelPlanning
-  /**
-   * current date/time in game
-   */
-  gameDate: string
   /** new orders templates for this player */
   templates: TemplateBody[]
   /** descriptions of platform types (used to generate icons) */
   platformTypes: PlatformTypeData[]
+  activityTimeChanel: (role: string, message: string) => void
+  saveMessage: (currentWargame: string, details: MessageDetails, message: any) => Function
+  saveNewActivityTimeMessage: (role: string, activity: PlainInteraction, dbName: string) => Function
+  dispatch: Dispatch<any>
+  state: PlayerUi
```

Comment:
@lebaphi - I'm having a problem with the `story` loading the state data.  But, I think I'd rather drop the `state` from props, and pass in the specific parameters (for and role).
Do you see any problem with this?

Reply:
it's okay @IanMayo, passing specific parameters will make the props list too long instead of putting them all in state

Reply:
I had a look, and `PlayerUi` wasn't referred to from anywhere in the `package/components`. It _seemed_ right to continue that practice :-)